index.html

```diff
@@ -260,6 +260,439 @@ <h1 class="primary-text1">
                 </li>
             </ul>
     </section>
+    <section class="work">
+        <ul class="main-card-container">
+            <li class="card">
+                <img class="snap-img-class" src="resources/Snapshoot Portfolio.svg" alt="resources/Snapshoot Portfolio.svg"></img>
+                <div class="card-block">
+                    <div class="card-header">
+                        <span class="card-header-text">Tonic</span>
+                        <div class="card-header-frame">
+                            <div class="card-header-frame-client">
+                                <span class="card-header-frame-client-text">CANOPY</span>
+                            </div>
+                            <span class="counter">
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                                </svg>
+                            </span>
+                            <div class="card-header-frame-role">
+                                <span class="card-header-frame-role-text">Back End Dev</span>
+                            </div>
+                            <span class="counter">
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                                </svg>
+                            </span>
+                            <div class="card-header-frame-year">
+                                <span class="card-header-frame-year-text">2015</span>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="card-description"> A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
```

Comment:
- [ ] This is the card paragraph and a `<p>` tag would be better appropriate in this case for better semantics. Kindly change that. (remember to make the changes in all project cards)